core/client/fs/src/main/java/alluxio/client/block/policy/options/GetWorkerOptions.java

```diff
@@ -53,7 +53,7 @@ public BlockInfo getBlockInfo() {
   /**
    * @return the list of block worker infos
    */
-  public Iterable<BlockWorkerInfo> getBlockWorkerInfos() {
+  public List<BlockWorkerInfo> getBlockWorkerInfos() {
```

Comment:
Questionable signature change, we previously modified the underlying object. Alternatively we could return something immutable and use the set API.

Reply:
Updated, thanks.